fixed locale for decimal point interpretation of ascii files
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 meshlab (1.3.3) precise; urgency=low
 .
   * new snapshot release
Author: Stefan Schlager <zarquon42@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- meshlab-1.3.3.orig/meshlab/src/meshlab/main.cpp
+++ meshlab-1.3.3/meshlab/src/meshlab/main.cpp
@@ -33,7 +33,7 @@ int main(int argc, char *argv[])
   QCoreApplication::setOrganizationName(MeshLabApplication::organization());
   QString tmp = MeshLabApplication::appArchitecturalName(MeshLabApplication::HW_ARCHITECTURE(QSysInfo::WordSize));
   QCoreApplication::setApplicationName(MeshLabApplication::appArchitecturalName(MeshLabApplication::HW_ARCHITECTURE(QSysInfo::WordSize)));
-		
+  std::setlocale(LC_NUMERIC, "C");  	
 		if(argc>1)	
 		{
 			QString helpOpt1="-h";
